.typespec/fine-tuning/custom.tsp

```diff
@@ -43,6 +43,18 @@ model FineTuningJobHyperparameters {
    * through the training dataset.
    */
   n_epochs: FineTuningJobHyperparametersNEpochsChoiceEnum | int32 = FineTuningJobHyperparametersNEpochsChoiceEnum.auto;
+
+  /**
+   * Number of examples in each batch. A larger batch size means that model parameters
+   * are updated less frequently, but with lower variance.
+   */
+  batch_size: FineTuningJobHyperparametersBatchSizeChoiceEnum | int32 = FineTuningJobHyperparametersBatchSizeChoiceEnum.auto;
+
+  /**
+   * Scaling factor for the learning rate. A smaller learning rate may be useful to avoid
+   * overfitting.
+   */
+  learning_rate_multiplier: FineTuningJobHyperparametersLearningRateMultiplierChoiceEnum | float32 = FineTuningJobHyperparametersLearningRateMultiplierChoiceEnum.auto;
```

Comment:
Great catch -- you're right, this should be removed.

Reply:
Wait, no, I take that back. The spec is wrong:
https://github.com/Azure/azure-sdk-assets/blob/f14093f58e6fea221278efc01637d41cef3d4220/net/.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/SessionRecords/FineTuningTests/JobsFineTuning.json#L30

Despite saying otherwise, FT responses *do* include these.